README.md

```diff
@@ -398,5 +399,26 @@ nav:
   search: /search/
 ```
 
+### Custom Github 404 page
+
+When you hosting your hexo blog use Github Page, if Visitors a not exist post page, it will be redicrt to a deafult github 404 page. about this , you can custom the 404 page.
+
+First, as you create 404 page in Hexo
+
+```sh
+$ hexo new page "404"
+```
+
+It will generates a new 404 folder and new index file at `source/404/index.md`, you can customize the 404 page .md file
+
+And, you need defining the permalink in the Frontmatter of `source/404/index.md`
```

Comment:
You also need to define a permalink in the front-matter of the page (`source/404/index.md`)